setuptools/dist.py

```diff
@@ -799,7 +819,7 @@ def _include_misc(self, name, value):
             )
         else:
             new = [item for item in value if item not in old]
-            setattr(self, name, old + new)
+            setattr(self, name, list(old) + new)
```

Comment:
By adding annotations to the method, mypy now raises this issue. This fixes a `TypeError` when a `Distribution`'s old included attribute was a `tuple`.